src/kernels/ipywidgets/types.ts

```diff
@@ -102,15 +96,6 @@ export interface ILocalResourceUriConverter {
      * ```
      */
     asWebviewUri(localResource: Uri): Promise<Uri>;
-    /**
-     * The converter will post an event when it needs to convert the webview URI
-     */
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    requestUri: Event<Uri>;
```

Comment:
Removed this event.
I personally think its simpler to read the code by firing a method to get information as opposed to firing an event to ask for information.
Events - fire and forget, and method (callbacks) mean we need a result.